src/serial-console/azext_serialconsole/custom.py

```diff
@@ -0,0 +1,730 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+import json
+import threading
+import sys
+import uuid
+import time
+import re
+import textwrap
+import numpy     # pylint: disable=unused-import
+import wsaccel   # pylint: disable=unused-import
+import websocket
+import requests
+from azure.cli.core.azclierror import UnclassifiedUserFault
+from azure.cli.core.azclierror import ResourceNotFoundError
+from azure.cli.core.azclierror import AzureConnectionError
+from azure.cli.core.azclierror import ForbiddenError
+from azure.core.exceptions import ResourceNotFoundError as ComputeClientResourceNotFoundError
+from azext_serialconsole._client_factory import _compute_client_factory
+
+
+# pylint: disable=too-few-public-methods
+# pylint: disable=too-many-instance-attributes
+class GlobalVariables:
+    def __init__(self):
+        self.webSocket = None
+        self.terminalInstance = None
+        self.serialConsoleInstance = None
+        self.terminatingApp = False
+        self.loading = True
+        self.firstMessage = True
+        self.blockPrint = False
+        self.trycount = 0
+        self.OSIsWindows = False
```

Comment:
You should use snake case instead of camel case for the names of property, variable and function in python.